packages/components/molecules/f-rating/src/tenants/es-ES.js

```diff
@@ -1,4 +1,6 @@
 export default {
     locale: 'es-ES',
-    text: 'I am a VRating Component (ES)'
+    ratings: {
+        starsDescription: '{rating} star out of {total} | {rating} stars out of {total}'
```

Comment:
should we change all languages other than English to something like "TODO - GET APPROPRIATE TRANSLATION" - otherwise it might get forgotten about